roaring/containers_btree.go

```diff
@@ -120,7 +122,7 @@ func (btc *bTreeContainers) Count() (n uint64) {
 	e, _ := btc.tree.Seek(0)
 	_, c, err := e.Next()
 	for err != io.EOF {
-		n += uint64(c.N())
+		n += uint64(c.n)
```

Comment:
wouldn't N() be inlined anyway?

Reply:
Yes, probably. Context: I had a previous patch where *every* `.n` had to be a `.N()`, and I ended up reverting these fairly aggressively. So this change isn't really necessary, but it's also harmless.